docs/subvolume.md

```diff
@@ -0,0 +1,46 @@
+# Subvolume cleanup
+
+The subvolume command is used to clean the stale subvolumes
+which have no parent-pvc attached to them.
+The command would list out all such subvolumes which needs to be removed.
+This would consider all the cases where we can have stale subvolume
+and delete them without impacting other resources and attached volumes.
+
+The subvolume command will require the following sub commands:
+* `ls` : [ls](#ls) lists all the subvolumes
+    * `--type=stale`: lists all the stale volume that can be deleted
+* `delete` : [delete](#delete) stale subvolumes after user's confirmation.
+    It will list and delete only the stale subvolumes to prevent any loss of data.
+    * It takes 3 arguments:
+      * list of subvolumes of same filesystem and subvolumegroup
+      * filesystem name
+      * subvolumegroup name
+
+## ls
+
+```bash
+kubectl rook-ceph subvolume ls
+
+# ocs-storagecluster-cephfilesystem csi-vol-427774b4-340b-11ed-8d66-0242ac110004 csi
```

Comment:
What are the three columns? <filesystem> <subvolume> <subvolumeGroup>? It would be helpful to show a header for that. Also, let's add a fourth column for `stale | in-use` so they can immediately see which ones are stale or in-use.

Reply:
The command has `Print the list of stale subvolumes no longer in use` as help text, so I assume all listed volumes are stale.

Reply:
> The command has `Print the list of stale subvolumes no longer in use` as help text, so I assume all listed volumes are stale.

yes, we don't need to mention the status, cause ls will print all subvolumes while --stale will print the stale ones, so I guess it should be clear.